data/core/help.cfg

```diff
@@ -285,18 +285,15 @@ Normally you can undo a unit’s movement, as long as an event with a randomized
         title= _ "Save-loading"
-        text= _ "Random numbers are part of Wesnoth, attacks can fail and units can die due to bad luck. This is an expected part of the game, and one of the aims of the game is that a sufficiently skilled player should be able to complete all of the mainline campaigns, on hard, without needing to save-load. Going back to a previous turn to try a different strategy is a part of learning the game, but we recommend against reloading merely to try the same strategy again while hoping for better luck." + _ "
+        text= _ "Random numbers are part of Wesnoth, attacks can fail and units can die due to bad luck — this is an expected part of the game. Going back to a previous turn to try a different strategy is a part of learning the game, but we recommend against reloading merely to try the same strategy again while hoping for better luck." + "
 
-One of the challenges of the game is to work out how to protect your heroes. Small risks quickly build up: if you have five important units, and they each have just a 1% chance of death each turn, you can <italic>text='expect'</italic> one of them to die about every 20 turns. So, you’ll rarely make it through a scenario without having it happen." + _ "
+" + _ "One of the challenges of the game is to work out how to protect your heroes. Small risks quickly build up: if you have five important units, and they each have just a 1% chance of death each turn, you can <italic>text='expect'</italic> one of them to die about every 20 turns. So, you’ll rarely make it through a scenario without having it happen." + "
 
-The following points are under discussion on the forums. At least for the 1.15.x series before 1.16.0 please assume that the forum threads are more authoritative than the text on this page." + _ "
+" + _ "Small risks build up for your non-hero units too — however these units don’t have to survive. While the story and dialogue usually aim to give you some emotional bond with your troops, the game mechanics reward cold-blooded thoughts such as “even if those units survive, I’m not going to recall them in any future scenarios, ...”, and the even colder “..., and that will reduce my upkeep”." + "
 
-In the mainline campaigns, on hard:
-• A sufficiently skilled player will have a good chance to win each scenario on the first time that they see that scenario, without foreknowledge of what will happen.
-• This assumes that the player picks up all of the clues given in the dialogue.
-• There are more experience points available than on easy, and the player will be able to train enough troops to cope with losing some L2 or L3 units." + _ "
+" + _ "The difficulty levels are balanced assuming that the player won’t save-load. In the mainline campaigns, harder difficulties generally have more enemies and thus more experience points available; this leads to them being balanced assuming that the player will be able to train enough troops to cope with losing some L2 or L3 units." + "
```

Comment:
I'd change L2 and L3 to "level two" and "level three", unless the former way is consistent with existing text elsewhere.